openrtb_ext/imp_bigoad.go

```diff
@@ -0,0 +1,6 @@
+package openrtb_ext
+
+type ExtImpBigoAd struct {
+	Host  string `json:"host"`
```

Comment:
@BIGOAds  `static/bidder-params/bigoad.json` has no description for `host` bidder param

Reply:
refer https://github.com/prebid/prebid-server/pull/3711/files#r1660582976

Bigoad adapter code does not need `host`. Can remove `host` from struct